src/ansys/dpf/core/operators/operator.mustache

```diff
@@ -89,6 +89,9 @@ class {{class_name}}(Operator):
                     {{/has_types}}
                     optional={{optional}},
                     document="""{{{document}}}""",
+                    {{#has_derived_class}}
```

Comment:
@MichaelNale are those changes needed ? I don't think this is useful, you other changes should fix the "mesh_info_provider_op.outputs.mesh_info()" call.